tailwind.config.mjs

```diff
@@ -4,6 +4,34 @@ export default {
 	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
 	theme: {
 		extend: {
+			colors: {
```

Comment:
Let's move it outside extend, so the only available colors are the one defined here please 🙏